fetch/main.js

```diff
@@ -1,17 +1,20 @@
 async function getData() {
-  try {
     const response = await fetch('https://jsonplaceholder.typicode.com/users');
+if (!response.ok){
+  new Error(`The reponse was unsucessful with the status Code - ${reponse.status}`)
+}
+  try {
     const data = await response.json();
     console.log(data);
   } catch (e) {
-    console.error(e);
+    console.error(Error);
```

Comment:
It was correct before. `console.error(Error)` doesn't do anything useful. Error is a class, not an object.

Looks like I missed this down below on the prior review. Sorry.